lib/SPIRV/libSPIRV/SPIRVNameMapEnum.h

```diff
@@ -182,10 +182,10 @@ template <> inline void SPIRVMap<Decoration, std::string>::init() {
 
   // From spirv_internal.hpp
   add(internal::DecorationCallableFunctionINTEL, "CallableFunctionINTEL");
-  add(internal::DecorationMathOpDSPModeINTEL, "MathOpDSPModeINTEL");
-  add(internal::DecorationInitiationIntervalINTEL, "InitiationIntervalINTEL");
-  add(internal::DecorationMaxConcurrencyINTEL, "MaxConcurrencyINTEL");
-  add(internal::DecorationPipelineEnableINTEL, "PipelineEnableINTEL");
+  add(DecorationMathOpDSPModeINTEL, "MathOpDSPModeINTEL");
+  add(DecorationInitiationIntervalINTEL, "InitiationIntervalINTEL");
+  add(DecorationMaxConcurrencyINTEL, "MaxConcurrencyINTEL");
+  add(DecorationPipelineEnableINTEL, "PipelineEnableINTEL");
```

Comment:
It says "from spirv_internal.hpp", but as I understood the decorations are now in spirv.hpp header. Should we move this to other place?
(same for capabilities)

Reply:
Thanks! I'll move the code